setting-up-cla-check.md

```diff
@@ -0,0 +1,62 @@
+# Setting up the CNCF CLA check
+
+If you are trying to sign the CLA so your PR's can be merged, please 
+[read the CLA docs](https://git.k8s.io/community/CLA.md)
+
+If you are a Kubernetes GitHub organization or repo owner, and would like to setup
+the Linux Foundation CNCF CLA check for your repositories, please read on.
+
+## Setup the webhook
+
+1. Go to the settings for your organization or webhook, and choose Webhooks from the menu, then
+   "Add webhook"
+    - Payload URL: https://identity.linuxfoundation.org/lfcla/github/postreceive?group=284&comment=no&target=https://identity.linuxfoundation.org/projects/cncf
+      - `group=284` specifies the ID of the CNCF project authorized committers group in our CLA system.
+      - `comment=no` specifies that our system should not post help comments into the pull request (since the Kubernetes mungebot does this).
+      - `target=https://identity.linuxfoundation.org/projects/cncf` specifies what will be used for the "Details" link in Github for this status check.
+    - Content Type: application/json
+    - Secret: Please contact one of [@idvoretskyi](https://github.com/idvoretskyi), [@foxish](https://github.com/foxish), or [@spiffxp](https://github.com/spiffxp)
```

Comment:
When I previously spoke with Ihor he said the POCs for this were himself and Chris Aniszczyk. Is this an updated list from that?

Reply:
@mattfarina That's fine, I'm handling the questions of CLA for Kubernetes on behalf of CNCF.

Reply:
Well, it was more that I know foxish and I have the secret (he's who ihor referred me to when I asked), and wanted to avoid a bottleneck here, but upon further thought it's probably wiser to keep that secret with them.